### PR TITLE
Add Example ServiceAccount

### PR DESCRIPTION
**Pull request by jy-kkkim**
Added a new example of the ServiceAccount function.

### CODE REVIEW

File: examples/service-account.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+//
+// MinIO Object Storage (c) 2021 MinIO, Inc.
```
Comment (vadmeste) [inline examples/service-account.go:2]: s/2021/2022

File: examples/service-account.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+//
```
Comment (vadmeste) [inline examples/service-account.go:1]: You need to add this
```
 //go:build ignore
 // +build ignore
```

so this example won't be compiled with go lint and others

File: examples/service-account.go (hunk 1 above)
Comment (vadmeste) [inline examples/service-account.go:2]: s/2021/2022

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**